src/test/scala/mesosphere/marathon/integration/AppDeployIntegrationTest.scala

```diff
@@ -19,9 +23,256 @@ class AppDeployIntegrationTest
   //clean up state before running the test case
   before(cleanUp())
 
-  def extractDeploymentIds(app: RestResult[AppDefinition]): Seq[String] = {
-    for (deployment <- (app.entityJson \ "deployments").as[JsArray].value)
-      yield (deployment \ "id").as[String]
+  test("create a simple app without health checks") {
+    Given("a new app")
+    val app = appProxy(testBasePath / "app", "v1", instances = 1, withHealth = false)
+
+    When("The app is deployed")
+    val result = marathon.createApp(app)
+
+    Then("The app is created")
+    result.code should be (201) //Created
+    extractDeploymentIds(result) should have size 1
+    waitForEvent("deployment_success")
```

Comment:
I guess the question is what the test scope is ;) It is an integration test...

Alternatively, you could query the mesos api and check if the task is marked as running or something like that.

But you wouldn't for example test the generation of environment variables like $PORT which is also pretty important.